privtrak.go

```diff
@@ -193,18 +193,19 @@ func (u *userStats) update(req *bittorrent.AnnounceRequest) *StatDelta {
 	return nil
 }
 
-func (u *userStats) removeExpired(peerLifetime time.Duration) ([]StatDelta) {
+func (u *userStats) removeExpired(peerLifetime time.Duration, id ID) []StatDelta {
```

Comment:
I added this ID parameter so that IDs can still be included in statdeltas that are emitted as a result of GC / seedtime flushing. Was this an oversight that the ID wasn't included prior to this, or am I missing something?

Reply:
Well, back then the removeExpired method didn't emit deltas :) So I think this is fine